README.md

```diff
@@ -20,43 +21,133 @@ This stack will deploy a php application on X Amazon EC2 instances behind an ALB
+|`front_ebs_optimized`|Whether the Instance is EBS optimized or not, related to the instance type you choose.|`bool`|`false`|`False`|
+|`front_type`|Type of instance to use for front servers|`-`|`t3.small`|`False`|
+|`keypair_name`|SSH keypair name to use to deploy ec2 instances.|`-`|`cycloid`|`False`|
+|`metrics_sg_allow`|Additionnal security group ID to assign to Cycloid workers. Goal is to allow monitoring server to query metrics.|`-`|`""`|`False`|
+|`private_subnets_ids`|Amazon subnets IDs on which create each components.|`array`|``|`True`|
+|`public_subnets_ids`|Amazon subnets IDs on which create each components.|`array`|``|`True`|
+|`rds_backup_retention`|The days to retain backups for. Must be between 0 and 35. When creating a Read Replica the value must be greater than 0|`-`|`7`|`False`|
+|`rds_database`|RDS database name.|`-`|`application`|`False`|
+|`rds_disk_size`|RDS disk size.|`-`|`10`|`False`|
+|`rds_engine`|RDS database engine to use.|`-`|`mysql`|`False`|
+|`rds_engine_version`|The version of the RDS database engine|`-`|`5.7.16`|`False`|
+|`rds_multiaz`|If the RDS instance is multi AZ enabled.|`bool`|`false`|`False`|
+|`rds_parameters`|Name of the RDS parameters group to use.|`-`|`default.mysql5.7`|`False`|
+|`rds_password`|RDS password. expected value is "${var.rds_password}" to get it from the pipeline.|`-`|`ChangeMePls`|`False`|
+|`rds_subnet_group`|RDS subnet group name to use. If not specified, create a dedicated group with private_subnets_ids.|`-`|``|`False`|
+|`rds_type`|RDS database instance size.|`-`|`db.t3.small`|`False`|
+|`rds_username`|RDS database username.|`-`|`application`|`False`|
+|`vpc_id`|Amazon VPC id on which create each components.|`-`|``|`True`|
+
+**Outputs**
+
+| Name | Description |
+|------|-------------|
+| alb_front_dns_name | DNS name of the front alb. |
+| alb_front_zone_id | Zone ID of the front alb. |
+| elasticache_address | Address of the elasticache. |
+| iam_s3-medias_user_key | Access key of the dedicated IAM user to access to the media S3 bucket. |
+| iam_s3-medias_user_secret | Access secret key of the dedicated IAM user to access to the media S3 bucket. |
+| rds_address | Address of the RDS database. |
+| rds_database | Database name of the RDS database. |
+| rds_port | Port of the RDS database. |
+| rds_username | Username of the RDS database. |
+| s3_medias | S3 bucket name dedicated to medias. |
+
+
+## Ansible
 
   * Playbook and packer config to build a debian image with telegraf, fluentd, nginx and php-fpm installed
 
+|Name|Description|Type|Default|Required|
+|---|---|:---:|:---:|:---:|
+|`cycloid_files_watched`|Provide log files you want to export to Cycloid logs.|`-`|`<Default log files watched>`|`False`|
+|`nginx_sites`|Contain Nginx vhosts to create on front servers. A default application and metrics vhosts are already provided.|`dict`|`<metric and application vhost>`|`False`|
+|`php_version_to_install`|PHP fpm version to install.|`-`|`7.2`|`False`|
```

Comment:
I assume all the tables of params for pipeline, terraform/ansible/ouptut was generated by your script? Or shall I check it too?

Reply:
yes tabs are generated by my scripts. From comments in the code